docs/evaluation/long-context.md

```
@@ -72,6 +72,12 @@ For more details see [https://github.com/NVIDIA/RULER/blob/rulerv2-ns](https://g
 - Benchmark is defined in [`nemo_skills/dataset/mrcr/__init__.py`](https://github.com/NVIDIA-NeMo/Skills/blob/main/nemo_skills/dataset/mrcr/__init__.py)
 - Original benchmark source is [here](https://huggingface.co/datasets/openai/mrcr).
 
+### graphwalks
+
+- Benchmark is defined in [`nemo_skills/dataset/graphwalks/__init__.py`](https://github.com/NVIDIA-NeMo/Skills/blob/main/nemo_skills/dataset/graphwalks/__init__.py)
+- Original benchmark source is [here](https://huggingface.co/datasets/openai/graphwalks).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)